nextflow_schema.json

```diff
@@ -12,17 +12,36 @@
             "description": "Define where the pipeline should find input data and save output data.",
             "required": ["outdir"],
             "properties": {
-                "input": {
+                "phyloplace_input": {
                     "type": "string",
                     "format": "file-path",
                     "exists": true,
-                    "schema": "assets/schema_input.json",
+                    "schema": "assets/schema_phyloplace_input.json",
                     "mimetype": "text/csv",
                     "pattern": "^\\S+\\.csv$",
-                    "description": "Path to comma-separated file containing information about the samples in the experiment.",
-                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with at least five columns, and a header row. See [usage docs](https://nf-co.re/phyloplace/usage#samplesheet-input).",
+                    "description": "Path to comma-separated file containing information about phylogenetic placements to perform.",
+                    "help_text": "This specifies parameters for direct phylogenetic placement of query sequences onto a reference phylogeny, in contrast to when a HMMER search is first performed, see `--phylosearch_input`. It has to be a comma-separated file with at least five columns, and a header row ('sample', 'queryseqfile', 'refseqfile', 'refphylogeny' and 'model'). See [usage docs](https://nf-co.re/phyloplace/usage#samplesheet-input).",
                     "fa_icon": "fas fa-file-csv"
                 },
+                "phylosearch_input": {
+                    "type": "string",
+                    "format": "file-path",
+                    "exists": true,
+                    "schema": "assets/schema_phylosearch_input.json",
+                    "mimetype": "text/csv",
+                    "pattern": "^\\S+\\.csv$",
+                    "description": "Path to comma-separated file containing information about phylogenetic searches to perform.",
+                    "help_text": "This specifies parameters for phylogenetic placement of unknown sequences onto reference phylogeny after first using HMMER to search through a collection of uknown sequences (see `--search_fasta`) in contrast to when only placing known query sequences, see `--phyloplace_input`. It has to be a comma-separated file with at least two columns that specifies search HMM profiles ('target' and 'hmm') and at least three columns specifying the reference phylogenies ('refseqfile', 'refphylogeny' and 'model') for placement of search results. (Decoy profiles, i.e. profiles only used for searches, not placements, are allowed.) See [usage docs](https://nf-co.re/phyloplace/usage#samplesheet-input).",
```

Comment:
In `schema_phylosearcg_input.json` its only
`"required": ["target", "hmm"]`
maybe that could be adjusted at some point.

Reply:
This is actually correct. I've made it possible to add hmm files without corresponding trees to act as baits. For instance, if we have GTDB trees for bacteria and archaea, one would add the eukaryotic and mitochondrial barrnap profiles as baits to make sure that these two are not placed incorrectly in either of the two GTDB trees. I'll make sure this is well documented.

Reply:
Yes documentation of such an awesome feature would be very helpful!